voila/app.py

```diff
@@ -618,9 +620,19 @@ def launch_browser(self):
             #     url = url_concat(url, {'token': self.token})
             url = url_path_join(self.connection_url, uri)
 
+            template_include_css = partial(include_css, self.voila_configuration.template, url)
+            template_include_js = partial(include_js, self.voila_configuration.template, url)
+            template_include_url = partial(include_url, self.voila_configuration.template, url)
```

Comment:
Can we move these operations into a function in `utils` which will return `template_include_css `, `template_include_js ` and `template_include_url `